### ISSUE TITLE
iOS: Strongbox opens when accepting AirDrop PDF

### ISSUE THREAD

**Issue by narkaTee** [strongbox-password-safe/Strongbox#433]
I recently noticed that strongbox seems to "hijack" PDF files when receiving them via AirDrop.
I was able to select where to save the pdf file, but after accepting the air drop strongbox instantly opens and tries to open the PDF file as a keepass file.

When I delete the Strongbox App iOS allows me to select to save the PDF File to the files app.

iOS version: 14.2

**Comment by strongbox-mark** [comment]
I don't get this behaviour on my own device. Could you try removing/reinstalling Strongbox to see if this has any effect? 

Remember to backup any local databases/key files etc before you do this.

If that doesn't help, I wonder if you wouldn't mind sharing a screen recording of this issue?

**Comment by narkaTee** [comment]
I already tried uninstalling and reinstalling Strongbox.

If strongbox is uninstalled there the "Open with" dialog opens, I can then choose to use the File App to save the PDF.
But as soon as I re-install it PDF go straight to Strongbox.

My guess is there is some new "which app can handle what file type" magic in iOS 14.
I found [this post](https://forums.macrumors.com/threads/airdrop-and-file-type-association-problem.2255707/?post=29274413#post-29274413
) which describes the Problem and possible workarounds.
The essence of the Post is:
> Actually, you need TWO or more apps that can read PDF documents.

If I install the Apple Books App (which can handle PDFs) I get the "Open with..." Dialog:
![IMG_2311](https://user-images.githubusercontent.com/12936343/99998666-53087a00-2dbf-11eb-8e40-a1566f84b230.PNG)
